### ISSUE TITLE
Better GSoC overview

### ISSUE THREAD

**Issue by cdeil** [OpenAstronomy/openastronomy.github.io#134]
In addition to what @bmorris3 suggested in #133 , I think it might be useful to have some overview of GSoC from previous years, especially have links or info for good previous GSoC projects somewhere discoverable for reference for potential students / mentors, but also for us to have an overview.

There is http://openastronomy.org/gsoc/ but as far as I can see that is only project ideas. There is no way to see which ones happened, who did them or info about the project. http://openastronomy.org/Universe_OA/ is nice to keep up to date with new blog posts, but again not a nice overview of previous projects.

It's probably be some work to collect that data and surface it in a good way, so definitely would suggest to focus on #133 first.

**Comment by bmorris3** [comment]
This isn't exactly what you're looking for, and this is very self-serving but if you like, we could create a "featured GSoC projects" section of the webpage, which might feature [this video I gave at pyastro about astroplan](https://www.youtube.com/watch?v=ML2zWwdZiQ0), which discussed what GSoC is, along with details about astroplan. I'm sure all of you have ideas of what other successful projects looked like, and how to "feature" them on the webpage.

**Comment by cdeil** [comment]
Here I meant creating an overview of all projects that actually happened, and there we can't attempt to try to highlight the "successful" ones, it's not possible / useful to make such an evaluation.

But I do think we should feature one or a few projects as good examples. Given that there's the video for astroplan I think it should definitely be used and a link to the video added. See my comment here: https://github.com/OpenAstronomy/openastronomy.github.io/issues/133#issuecomment-359898027

I just remembered that Numfocus did bloc posts highlighting and summarising their GSoC projects like this:
* https://www.numfocus.org/2017-gsoc-students-complete-work-on-numfocus-projects/
* https://www.numfocus.org/meet-our-gsoc-students-part-4-astropy-sunpy-shogun/

Basically what I was suggesting for openastro for previous years was to have something like https://www.numfocus.org/2017-gsoc-students-complete-work-on-numfocus-projects/

**Comment by bsipocz** [comment]
Yeah, I would be a bit worry to highlight a few projects only, it would actually be very nice to collect all projects that got finished (maybe including the patch being submitted to google, too).
